SixTrack/sixtrack.s

```diff
@@ -22351,16 +22351,16 @@ C Should get me a NaN
      &'|',f11.6,'|',11x,'|',11x,'|',11x,'|',11x,'|')
 10090 format('|',6x,'|',8x,'|',12x,'|','S','|',12x,'|',f12.6,'|', f13.7,&
      &'|',f11.6,'|',11x,'|',11x,'|',11x,'|',11x,'|')
-10100 format('|',6x,'|',8x,'|',12x,'|','S','|',f12.7,'|',f12.6,'|', f13.&
-     &7,'|',f11.6,'|',f11.7,'|',f11.7,'|',f11.7,'|',f11.7,'|')
+10100 format('|',6x,'|',8x,'|',12x,'|','S','|',f12.7,'|',ES12.6,'|',    &
+     &f13.7,'|',f11.6,'|',f11.7,'|',f11.7,'|',f11.7,'|',f11.7,'|')
 10110 format(/t10,'CO-TRACKING ENDED ABNORMALLY'/t10, 'PARTICLE NO. '   &
-     &,i7,' AT ELEMENT ',i4/ t10,'HORIZ:  AMPLITUDE = ',f15.3,          &
-     &'   APERTURE = ',f15.3/ t10,'VERT:   AMPLITUDE = ',f15.3,         &
+     &,i7,' AT ELEMENT ',i4/ t10,'HORIZ:  AMPLITUDE = ',ES23.16,        &
+     &'   APERTURE = ',f15.3/ t10,'VERT:   AMPLITUDE = ',ES23.16,       &
```

Comment:
Since you're changing F -> ES here, how small are these numbers typically? Could you post an example pre- and post changes?